src/pages/EmployeeView/EmployeeView.tsx

```diff
@@ -0,0 +1,115 @@
+import { getDateView, getWorkExp } from "../../utils/helper.ts";
+import DetailsSection from "../../components/Details/Details.tsx";
+import ActiveChip from "../../components/ActiveChip/ActiveChip.tsx";
+import { FieldValues } from "react-hook-form";
+
+const EmployeeView = ({
+  employee,
+}: {
+  employee: IAppEmployee | FieldValues;
+}) => {
+  return (
+    <>
+      {" "}
+      <div className="flex employee-intro-section">
+        <img
+          src={
+            employee.photoId === "" || employee.photoId === undefined
+              ? DummyImg
+              : typeof employee.photoId === "string"
+              ? employee.photoId
+              : URL.createObjectURL(employee.photoId[0])
+          }
+          alt="Employee image"
+        />
+        <div className="flex employee-intro">
+          <h2>{employee.firstName + " " + employee.lastName}</h2>
+          <div className="employee-status">
+            <ActiveChip isActive={employee.isActive === "Yes" ? true : false} />
+            <span>{employee.role.label}</span>
+          </div>
+          <div className="flex">
+            <div className="employee-info">
+              <DetailsSection
+                icon="person"
+                title="Department"
+                content={employee.department.label}
+              />
+              <DetailsSection
+                icon="person"
+                title="Date hired"
+                content={getDateView(employee.dateOfJoining!)}
+              />
+            </div>
+            <div className="employee-info">
+              <DetailsSection icon="mail" content={employee.email!} />
+              <DetailsSection icon="phone_iphone" content={employee.phone!} />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="employee-details-section">
+        <h2>Personal Details</h2>
+        <div className="detail-element">
+          <DetailsSection
+            title="Full Name"
+            content={employee.firstName + " " + employee.lastName}
+            matches
+            newline={true}
+          />
+          <DetailsSection
+            title="Date of Birth"
+            content={getDateView(employee.dob!)}
+            matches
+            newline={true}
+          />
+          <DetailsSection
+            title="Address"
+            content={employee.address!}
+            matches
+            newline={true}
+          />
+        </div>
+      </div>
+      <div className="employee-details-section">
+        <h2>Professional Details</h2>
+        <div className="detail-element">
+          <DetailsSection
+            title="Role"
+            content={employee.role.label}
+            matches
+            newline={true}
+          />
+          <DetailsSection
+            title="Designation"
+            content={employee.designation!}
+            matches
+            newline={true}
+          />
+          <DetailsSection
+            title="Work Experience"
```

Comment:
convert to year and months. if the work experience exceeds 12 months show it as 1 year